doc/news/changes.h

```diff
@@ -38,6 +38,12 @@ inconvenience this causes.
 </p>
 
 <ol>
+  <li> Changed: GridGenerator::hyper_rectangle and GridGenerator::subdivided_hyper_rectangle
+  now take points with @p dim components to correctly handle cases with codim1 and codim2.
```

Comment:
"codim1" and "codim2" are a bit developer-speak. I don't now how clearly users would understand that. How about "...to correctly handle cases of meshes embedded in higher-dimensional spaces"?